views/event-form.hbs

```diff
@@ -0,0 +1,186 @@
+{{#if edit}}
+  <form class="add-form" action="/{{lang}}/edit-event/{{event._id}}" method="post">
+    <h1>{{localText.edit}}</h1>
+    <input type="hidden" name="_method" value="put">
+{{else}}
+  <form class="add-form" action="/{{lang}}/add-event" method="post">
+    <h1>{{localText.add}}</h1>
+    <input type="hidden" name="_method" value="post">
+{{/if}}
+
+<span>{{localText.formInfo}}</span>
+<fieldset name="en" class="langFieldset">
+  <legend>{{localText.english}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.local.name}}
+      <input type="text" name="name_en" value="{{event.local.name}}">
+    {{else}}
+      <input type="text" name="name_en">
+    {{/if}}
+  </label>
+
+  <label>{{localText.description}}
+    {{#if event.local.description}}
+      <input type="text" name="description_en" value="{{place.local.description}}">
+    {{else}}
+      <input type="text" name="description_en">
+    {{/if}}
+  </label>
+</fieldset>
+
+<fieldset name="en" class="langFieldset">
+  <legend>{{localText.arabic}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.local.name}}
```

Comment:
so I've been trying to work this out, but I think this is where the problem comes from of when you edit you get the english details in the arabic fields. If you're at `/en/edit-place` then `event.local.name` here will refer to the english name, not the arabic.

As this fieldset is always gonna be specific to arabic instead of `event.local.name` you should use `event.ar.name` and in the english one you should use `event.en.name` (both obviously if they exist)

Also the same applies to the `description` and the also in another PR we need to do this to fix the places.